Game/Assets/Coliseo/Scripts/BlockingTrue.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class BlockingTrue : StateMachineBehaviour {
+
+	 // OnStateEnter is called when a transition starts and the state machine starts to evaluate this state
+	override public void OnStateEnter(Animator animator, AnimatorStateInfo stateInfo, int layerIndex) {
```

Comment:
I'll open all of the scripts and find/replace all tabs with spaces. I usually assume they aren't there so I feel better about myself.